superset/dashboards/filters_state/dao.py

```diff
@@ -0,0 +1,93 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import logging
+from typing import Optional
+
+from sqlalchemy.exc import SQLAlchemyError
+
+from superset.dao.exceptions import (
+    DAOCreateFailedError,
+    DAODeleteFailedError,
+    DAOUpdateFailedError,
+)
+from superset.dashboards.dao import DashboardDAO
+from superset.extensions import cache_manager
+from superset.key_value.utils import cache_key
+
+logger = logging.getLogger(__name__)
+
+
+class KeyValueDAO:
```

Comment:
This DAO is basically a set of commands as they're not really talking to the DB, other than to delegate to other DAOs. Pls break these out into CMDs instead.